frontend/controller/backend.js

```diff
@@ -13,6 +15,11 @@ import { PUBSUB_INSTANCE } from './instance-keys.js'
 // const nacl = require('tweetnacl')
 import nacl from 'tweetnacl'
 
+// Used in the 'backend/translations/get()' SBP function.
```

Comment:
`get()` shouldn't have a `()`, this should read simply:

```
// Used by 'backend/translations/get'
```

Reply:
Ok :)